src/diffusers/hooks/layerwise_casting.py

```diff
@@ -134,6 +167,7 @@ def apply_layerwise_casting(
         skip_modules_classes,
         non_blocking,
     )
+    _disable_peft_input_autocast(module)
```

Comment:
The method is already version-guarded. So, no worries. 

(For other reviewers).